### ISSUE TITLE
Python version when installing Mujoco

### ISSUE THREAD

**Issue by lrozo** [robotlearn/pyrobolearn#12]
As the installation instructions suggest to have either 2.7 or 3.5, the Mujoco installation fails as Mujoco 2.0 is supposed for python >=3.6 . Therefore, we need to mention this when installing Mujoco as an alternative simulator. Another solution would be to force the `pip install`  with 3.6 version.

**Comment by bdelhaisse** [comment]
Yes, they changed that on Aug 15: see following commit https://github.com/openai/mujoco-py/commit/0711ab58777a28aff847adbf05ba246a337908a0
On my computer, I have the repo set to a previous commit https://github.com/openai/mujoco-py/commit/9ca102f0a3cd744014273191d7f79d0e9fff24ee

Currently, it is not clear to me why they changed to Python 3.6 (I assume there is a code where they use a specific syntax or library); I will have to look more carefully at each commit after the one that I am using. However, if you want you can try to modify their `setup.py` by replacing `python_requires='>=3.6'` to `python_requires='>=3.5'`.
